decide/src/main/java/group17/LicAnalyzer.java

```diff
@@ -75,10 +75,70 @@ public boolean lic9(InputHandler input) {
+     * seperated by quantity2 with area greater than AREA1.
+     *
+     * @param quantity1 Number of consecutive intervening points between point 1 and point 2
+     * @param quantity2 Number of consecutive intervening points between point 2 and point 3
+     * @param input Class holding relevant data
+     *
+     * @return boolean True if such points exists, false otherwise
+     */
+    private boolean helperLic10ThreePointsSeperatedBy(int quantity1, int quantity2, InputHandler input)
+    {
+        for (int i = quantity1 + quantity2 + 2; i < input.NUMPOINTS; ++i) {
+            int vertex3 = i;
+            int vertex2 = i - quantity1 - 1;
+            int vertex1 = i - quantity1 - quantity2 - 2;
+
+            //Calculate triangle area from three points using shoelace formula
+            double xCoordinate1 = input.X_COORD[vertex1];
+            double yCoordinate1 = input.Y_COORD[vertex1];
+
+            double xCoordinate2 = input.X_COORD[vertex2];
+            double yCoordinate2 = input.Y_COORD[vertex2];
+
+            double xCoordinate3 = input.X_COORD[vertex3];
+            double yCoordinate3 = input.Y_COORD[vertex3];
+
+            double triangleArea = 0.5 * Math.abs( (xCoordinate1 - xCoordinate3) * (yCoordinate2 - yCoordinate1)
+                                                 -(xCoordinate1 - xCoordinate2) * (yCoordinate3 - yCoordinate1)
+                                                );
+            if (triangleArea > input.AREA1) {
+                return true;
+            }
+        }
+
         return false;
     }
 
+    /*
+     * Evaluates existence of a set of three points seperated by E_PTS and F_PTS
+     * respectively that together has area greater than AREA1.
+     *
+     * @param  input  object whose members hold data for the problem; relevant 
+                      fields for lic10 are: NUMPOINTS, E_PTS, F_PTS, AREA1, X_COORD, Y_COORD
+     * @return        true if set exists, false otherwise
+     */
+    public boolean lic10(InputHandler input) {
+        if (input.NUMPOINTS < 5 || input.NUMPOINTS > 100) {
+            throw new IllegalArgumentException("Exception thrown from: LIC 10. Reason: NUMPOINTS outside range [5, 100].");
+        } else if (input.E_PTS < 1 || input.F_PTS < 1) {
+            throw new IllegalArgumentException("Exception thrown from: LIC 10. Reason: E_PTS < 1 or F_PTS < 1.");
+        } else if (input.NUMPOINTS < input.E_PTS + input.F_PTS + 3) {
+            throw new IllegalArgumentException("Exception thrown from: LIC 10. Reason: NUMPOINTS < E_PTS + F_PTS + 3.");
+        }
+
+        boolean pointsFound = helperLic10ThreePointsSeperatedBy(input.E_PTS, input.F_PTS, input);
+        if (!pointsFound) {
+            // Points might exist but seperated in opposite order to call above
+            pointsFound = helperLic10ThreePointsSeperatedBy(input.F_PTS, input.E_PTS, input);
+        }
+        return pointsFound;
```

Comment:
This can be changed to this

`return helperLic10ThreePointsSeperatedBy(input.E_PTS, input.F_PTS, input) || helperLic10ThreePointsSeperatedBy(input.F_PTS, input.E_PTS, input);`